surveyor/surveyor.go

```diff
@@ -11,7 +11,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package surveyor is used to garner data from a NATS deployment for Prometheus
```

Comment:
prefer we leave it here and not add an empty doc file personally, though this pattern is common enough that it's probably fine.  So LGTM.

Reply:
Makes sense to me. Pushed a commit for this.